dev/tests/integration/testsuite/Magento/Bundle/_files/order_with_bundle_shipped_together_rollback.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\TestFramework\Workaround\Override\Fixture\Resolver;
+
+Resolver::getInstance()->requireDataFixture(
+    'Magento/Bundle/_files/order_with_bundle_shipped_separately_rollback.php'
```

Comment:
Maybe it would be better to not depend on this rollback because it can be changed in the future and will become not suitable here?

Reply:
done